core/textile/bucket_factory.go

```diff
@@ -200,7 +200,7 @@ func (tc *textileClient) createBucket(ctx context.Context, bucketSlug string) (B
 
 	// We store the bucket in a meta thread so that we can later fetch a list of all buckets
 	log.Debug("Bucket " + bucketSlug + " created. Storing metadata.")
-	schema, err := m.CreateBucket(ctx, bucketSlug, dbID.String())
+	schema, err := m.CreateBucket(ctx, bucketSlug, utils.CastDbIDToString(*dbID))
```

Comment:
🤦🏻 

I started with `utils.CastDbIDToString()` and later changed it to `dbID.String()` because it looked more intuitive.